### PR TITLE
add: flashrom-cros

### PR DESCRIPTION
**Pull request by WeirdTreeThing**

### CODE REVIEW

File: anda/tools/flashrom-cros/flashrom-cros.spec [hunk 1]
```diff
@@ -0,0 +1,37 @@
+%global commit  c1ab7468d28d164a30d598eb3e42a5febaf73bbc
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:		flashrom-cros
+Version:	%shortcommit
```
Comment (madonuko) [inline anda/tools/flashrom-cros/flashrom-cros.spec]: please put `%commit_date` before `%shortcommit`

File: anda/tools/flashrom-cros/flashrom-cros.spec [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Release:	1%{?dist}
+Summary:	Simple program for reading/writing flash chips content (ChromiumOS fork)
+License:	GPL-2.0-only
+URL:		https://chromium.googlesource.com/chromiumos/third_party/flashrom
+Source0:	%url/+archive/refs/heads/release-R130-16033.B.tar.gz
+BuildRequires:	gcc gnupg2 libusb1-devel meson pciutils-devel python3-sphinx systemd zlib-devel dmidecode
+Requires:	dmidecode
+Requires:	udev
```
Comment (madonuko) [inline anda/tools/flashrom-cros/flashrom-cros.spec]: `Packager: name <mail@example.com>`

File: anda/tools/flashrom-cros/flashrom-cros.spec [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+flash chips. It is designed to flash BIOS/EFI/coreboot/firmware/optionROM
+images on mainboards, network/graphics/storage controller cards, and various
+other programmer devices.
+ 
+%prep
+%setup -C
+ 
+%build
+meson -Dtests=disabled -Ddocumentation=disabled -Duse_internal_dmi=false . build
+meson compile -C build
```
Comment (madonuko) [inline anda/tools/flashrom-cros/flashrom-cros.spec]: can you use the `%meson -Dtests=disabled -Documentation=blah blah blah` and `%meson_build` macros instead?

**Review by Owen-sz — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2446)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2447)|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2448)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11558429962) for details

**Pull request merged**